lib/rbeapi/api/staticroutes.rb

```diff
@@ -60,29 +61,36 @@ class Staticroute < Entity
       def getall
         regex = /
           (?<=^ip\sroute\s)
-          ([^\s]+)\s                # captures network
-          ([^\s$]+)                 # captures next hop
-          (?:\s\d+)                 # non-capture metric
-          (?:\stag\s\d+)            # non-catpure route tag
+          ([^\s]+)\s                # capture destination
+          ([^\s$]+)                 # capture next hop IP or egress interface
+          [\s|$](\d+)               # capture metric (distance)
+          [\s|$]{1}(?:tag\s(\d+))?  # catpure route tag
```

Comment:
The ?:tag is a named capture right? Is this being used anywhere? If not, drop it.